### DISCUSSION TITLE
html5 web client

### DISCUSSION THREAD

**Discussion by drago73** [evennia/evennia#2864] [General Discussion]
hi, is possible use in web client images or sounds?

**Comment by Antrare** [comment]
Did you read the documentation?  Check out this, https://www.evennia.com/docs/1.0-dev/Components/Webclient.html and you can see there's a multimedia plugin.  

_multimedia.js An basic plugin to allow the client to handle “image” “audio” and “video” messages from the server and display them as inline HTML._

**Reply by CharlesMccown** [reply]
Wow, and another cool discovery. It's almost like every time I learn something new about Evennia, I love it more.

Is this only in 1.0?

The .95 documentation on web client seems pretty empty, so maybe I made an error in starting my learning journey at the current version.

**Reply by VexiaOnline** [reply]
Is it indeed in the 0.9.5 webclient. Take a look in-game in the options, and you can also see the plugin in the plugins directory. :)

Sending them is as easy as tagging the outgoing msg text with a type!
